src/components/PopupWithForm.js

```diff
@@ -22,11 +22,9 @@ export default class PopupWithForm extends Popup {
 
   setEventListeners() {
     super.setEventListeners();
-    this._popupForm.addEventListener("submit", this._handleSubmit);
```

Comment:
Needs correction: this line was correct, you should add the method by reference, because you remove the listener when closing the popup